arangod/Aql/LazyConditions.cpp

```diff
@@ -0,0 +1,205 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2014-2024 ArangoDB GmbH, Cologne, Germany
+/// Copyright 2004-2014 triAGENS GmbH, Cologne, Germany
```

Comment:
The copyright note looks a bit outdated 😁

Reply:
It is actually a current one, including a reference to the "Business Source License" a few lines below.

Reading the copyright years, I can change it to "Copyright 2024 ArangoDB GmbH, Cologne, Germany" only. I don't know if this is legally correct or if for some reason we need to retain the reference to triAGENS in there.
My personal preference would be to remove the reference to the years entirely, so we could unify the header across all files and don't need to bother about it when a new year begins etc.